builtin/init.lua

```diff
@@ -7,6 +7,7 @@
 
 -- Initialize some very basic things
 function core.debug(...) core.log(table.concat({...}, "\t")) end
+print = core.debug -- Override native print and use our logging system
```

Comment:
Nope, print isn't logged to debug.txt right now. Perhaps it was before the logging PR, but not now. Right now it seems that print directly outputs to stdout, which leads the console to "freak out"; e.g. you won't see your own input anymore. Therefore this is needed.